satellite/gracefulexit/endpoint.go

```diff
@@ -295,7 +303,12 @@ func (endpoint *Endpoint) doProcess(stream processStream) (err error) {
 			}
 
 			var transferMsg *pb.SatelliteMessage
+			mon.IntVal("graceful_exit_final_pieces_failed").Observe(progress.PiecesFailed)
+			mon.IntVal("graceful_exit_final_pieces_succeess").Observe(progress.PiecesTransferred)
+
 			processed := progress.PiecesFailed + progress.PiecesTransferred
+			mon.IntVal("graceful_exit_successful_bytes_transfer_ratio").Observe(progress.PiecesTransferred / processed)
```

Comment:
I think you want bytes transferred here, not pieces transferred

Reply:
It looks like we don't have a value for the total number of bytes that should have been transferred, so I don't think we can get this ratio. I changed it to `graceful_exit_successful_pieces_transfer_ratio` but I am not sure if we can get the bytes ratio with the data in the `progress` table. @benjaminsirb